AUTHORS

```diff
@@ -107,7 +107,7 @@ Ivar Lazzaro                            ivarlazzaro@gmail.com
 Jaff Cheng                              jaff.cheng.sh@gmail.com
 Jaime Caamaño Ruiz                      jcaamano@suse.com
 Jan-Erik Rediger                        janerik@fnordig.de
-Jarno Rajahalme                         jarno@covalent.io
+Jarno Rajahalme                         jarno@isovalent.com
```

Comment:
Given this is auto-generated, you might need to update https://github.com/cilium/cilium/blob/master/.mailmap instead.